### PR TITLE
Change wording in custom-configuration.md

### PR DESCRIPTION
**Pull request by duozersk**

### CODE REVIEW

File: docs/content/stack/custom-configuration.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ aliases:
 ---
 ## Custom Configuration {#custom-configuration}
 
-Custom configurations are useful when you have a larger or more complex project. One where a CI server is involved 
-or many people are on a project team, and you have to be careful about maintaining software versions. 
+Custom configurations are useful when you have a larger or more complex project. Once a CI server is involved 
+or many people are on a project team then you have to be careful about maintaining software versions.
```
Comment (shelane) [inline docs/content/stack/custom-configuration.md:10]: ...team, then...

**Comment by lmakarov** [comment]
@duozersk my applause for your Docksal Docs review marathon 👏  All PRs merged!
Grouping PRs (e.g., per file) will make it less involved to review next time 😉

**Comment by duozersk** [comment]
@lmakarov thank you for quick merges ;) Yeah, indeed, it would be cool to group them at least by file... It was just very convenient to notice a typo, click the "Edit this page" link and submit a fix. Grouping them would require more efforts ;)
Do we have a way to compile the docs on local machine? Like ready to use commands/anything to get the Hugo running locally with local link to view the compiled docs?

**Comment by lmakarov** [comment]
Instructions for working with Docksal docs locally are here:
https://github.com/docksal/docksal/tree/develop/docs

**Pull request merged**